src/m365/entra/commands/multitenant/MultitenantOrganization.ts

```diff
@@ -0,0 +1,7 @@
+export interface MultitenantOrganization {
+  createdDateTime?: string;
+  displayName?: string;
+  description?: string;
+  id?: string;
+  state?: string;
+}
```

Comment:
Is the `?` required? Properties like id and state seems to be available all the time. For other properties, they can be `null`, but none can be `undefined` right?

Reply:
I've noticed that all models from `@microsoft/microsoft-graph-types` have properties with `?`, so I've decide to do the same here.

Reply:
I suggest we use the `?` only when it can be undefined.